internal/server/api.go

```diff
@@ -189,7 +189,7 @@ func (service *Service) postEventHandler(httpRespW http.ResponseWriter, httpReq
 	err = json.Unmarshal(payLoad, &pEvent)
 	if err != nil {
 		http.Error(httpRespW, fmt.Sprintf("%v", err), http.StatusBadRequest)
-		log.Printf("Bad Request: %v", err)
+		log.Printf("Failed to unmarshall request body: %v", err)
```

Comment:
Tiny typo :) `unmarshall` should be `unmarshal`.

Reply:
True, thanks! I have also edited some more of the messages as they should not be capitalized at the beginning.